HfsChargesContainer/Helpers/RuntimeEnvVarsHandler.cs

```diff
@@ -0,0 +1,68 @@
+using Amazon;
+using Amazon.SimpleSystemsManagement;
+using Amazon.SimpleSystemsManagement.Model;
+using HfsChargesContainer.Helpers.Interfaces;
+using HfsChargesContainer.Options;
+
+namespace HfsChargesContainer.Helpers
+{
+    public class RuntimeEnvVarsHandler : IRuntimeEnvVarsHandler
+    {
+        private readonly IAmazonSimpleSystemsManagement _ssmClient;
+        private readonly Dictionary<string, string> _ssmToAppNameLookup;
+
+        public RuntimeEnvVarsHandler(
+            string environment,
+            IAmazonSimpleSystemsManagement? ssmClient = null,
+            Dictionary<string, string>? ssmToAppNameLookup = null
+        )
+        {
+            _ssmClient = ssmClient ?? new AmazonSimpleSystemsManagementClient(RegionEndpoint.EUWest2);
```

Comment:
`_ssmClient = ssmClient ?? new AmazonSimpleSystemsManagementClient(RegionEndpoint.EUWest2);`
Do you really need a null check here?

Reply:
It depends on how you'd want to structure this.
If you'd want this Client to be newed up within startup, then this check is not needed & the null default argument is not needed as well.

Reply:
I figured, the RuntimeEnvVarsHandler is specifically dependent on this particular client, so it might be simpler to new up it's instance within the constructor of this class itself. Though, for it to be testable, we need to give the constructor a slot to inject a mock.